src/test/apiTest/resources/application.properties

```diff
@@ -65,3 +65,4 @@ nocNotification.template.tribunal.id=${NOC_NOTIFICATION_TEMPLATE_TRIBUNAL_ID:tes
 
 assign_case_access_api_url=${AAC_URL:http://localhost:4454}
 apply_noc_access_api_assignments_path=/noc/apply-decision
+hmcts_service_id=BHA1
```

Comment:
can't see it being added to application-dev.properties?

Reply:
this will never change for local dev, theres no point overriding in application-dev